### PR TITLE
docs(newsletter): clean up formatting and verbiage of existing documentation

### PR DESCRIPTION
**Pull request by xelaint**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/graycoreio/daffodil/blob/develop/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Part of: #2694 

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information

### CODE REVIEW

**Comment by xelaint** [comment]
@griest024 there seemed to be some duplicating information so I tried my best to clean it up and remove what I thought was redundant information. Can you review and make sure I didn't remove important information?

File: libs/newsletter/README.md [hunk 1]
```diff
@@ -1,83 +1,39 @@
...
+talk about supported platforms -->
 
-## Getting Started
+## Installation
+To install `@daffodil/newsletter` and its dependencies, use the following commands in the terminal.
 
-This overview assumes that you have already set up an Angular project and have gone through the [Newsletter installation guide](/libs/newsletter/guides/installation.md). If you have not, we recommend you do that first.
+Install with npm:
+```
+npm install @daffodil/newsletter @daffodil/core @ngrx/store @ngrx/effects --save
```
Comment (damienwebdev) [inline libs/newsletter/README.md]: Remove extraneous packages.

File: libs/newsletter/README.md [hunk 2]
```diff
@@ -1,83 +1,39 @@
...
+```
 
-## Setting up your AppModule
+Install with yarn:
+```
+yarn add @daffodil/newsletter @daffodil/core @ngrx/store @ngrx/effects
```
Comment (damienwebdev) [inline libs/newsletter/README.md]: Remove extraneous packages.

File: libs/newsletter/README.md [hunk 3]
```diff
@@ -1,83 +1,39 @@
...
+```
 
-To get started, import the `StoreModule` and the `DaffNewsletterModule` at the top of your app.module file.
+> After installing, an ecommerce platform driver needs to be set up. We highly recommend installing the [in-memory web api](./guides/drivers.md) for fast, out-of-the-box development.
```
Comment (damienwebdev) [inline libs/newsletter/README.md]: Remove the "ecommerce" part of this.

Comment (damienwebdev) [inline libs/newsletter/README.md]: > We highly recommend installing the [in-memory web api](./guides/drivers.md) for fast, out-of-the-box development.

Should probably be

> We highly recommend installing the [in-memory web api](./guides/drivers.md) for getting started quickly.

File: libs/newsletter/README.md [hunk 4]
```diff
@@ -1,83 +1,39 @@
...
-}
-```
-
-> In this example, three observable streams are assigned from `newsletterFacade`. Then when `submitData` is called, the `newsletterFacade` will call its `dispatch` function which will send your data off to the backend and update the three observable streams.
-
-## Live Demo
-
-[Checkout a live example of the `DaffNewsletter` library in action!](https://stackblitz.com/edit/daff-newsletter-example)
+## Live demo
+Check out a [live example](https://stackblitz.com/edit/daff-newsletter-example) of `@daffodil/newsletter` in action!
```
Comment (damienwebdev) [inline libs/newsletter/README.md:39]: This demo needs an update desperately.

**Review by damienwebdev — CHANGES_REQUESTED** [review]

File: libs/newsletter/README.md (hunk 4 above)
Comment (xelaint) [reply libs/newsletter/README.md:39]: created an issue for this #3062

File: libs/newsletter/README.md (hunk 3 above)
Comment (xelaint) [reply libs/newsletter/README.md]: updated

Comment (xelaint) [reply libs/newsletter/README.md]: removed

File: libs/newsletter/README.md (hunk 2 above)
Comment (xelaint) [reply libs/newsletter/README.md]: removed

File: libs/newsletter/README.md (hunk 1 above)
Comment (xelaint) [reply libs/newsletter/README.md]: removed

**Pull request merged**